pep-0695.rst

```diff
@@ -951,20 +955,21 @@ TypeScript
 ----------
 
 TypeScript uses angle brackets to declare type parameters and for 
-specialization. The "extends" keyword is used to specify a bound. It can be
-combined with other type operators such as "keyof".
+specialization. The ``extends`` keyword is used to specify a bound. It can be
+combined with other type operators such as ``keyof``.
 
 TypeScript uses declaration-site variance. Variance is inferred from
-usage, not specified explicitly. TypeScript 4.7 will introduce the ability
-to specify variance using "in" and "out" keywords. This was added to handle
-extremely complex types where inference of variance was expensive.
+usage, not specified explicitly. TypeScript 4.7 introduced the ability
+to specify variance using ``in`` and ``out`` keywords. This was added to handle
+extremely complex types where inference of variance was expensive,
+yet the maintainers state that is useful for increasing readability.
```

Comment:
Readability was not the motivation for adding this capability. I meet often with the TypeScript team, and I'm very familiar with their motivations and the debates that led to their (reluctant) decision to add this capability.

Reply:
It wasn't a motivation, but my edit here was based on information from [the documentation](https://devblogs.microsoft.com/typescript/announcing-typescript-4-7-beta/#:~:text=one%20reason%20is%20that%20it%20can%20be%20a%20useful%20for%20a%20reader%20to%20explicitly%20see%20how%20a%20type%20parameter%20is%20used%20at%20a%20glance.)

Reply:
Could you provide some references for these reasons? Reading the pr https://github.com/microsoft/TypeScript/pull/48240, it seems to lead with positive motivations and nothing negative that I can see:
> Indeed, when generic type instantiations are related structurally, variance annotations serve no purpose. This is why TypeScript strictly doesn't need variance annotations. However, variance annotations are useful to assert desired type relations of their subject generic types.

Reply:
This information comes from direct conversations with Anders Hejlsberg, Ryan Cavanaugh, and Daniel Rosenwasser, the leads on the TypeScript team. I've discussed the arguments you've raised about variance. Here's how Daniel summarized their advice: 
"our perspective is that ideally we wouldn't have added [explicit] variance if we didn't need to"